app/src/main/java/org/simple/clinic/newentry/PatientEntryScreenController.kt

```diff
@@ -238,28 +183,14 @@ class PatientEntryScreenController @Inject constructor(
   }
 
   private fun showValidationErrorsOnSaveClick(events: Observable<UiEvent>): Observable<UiChange> {
-    val phoneNumberChanges = events.ofType<PatientPhoneNumberTextChanged>().map { it.phoneNumber }
-    val noPhoneToggles = events.ofType<PatientNoPhoneNumberToggled>().map { it.noneSelected }
-
-    val phoneNumberErrors: Observable<List<PatientEntryValidationError>> = events
-        .ofType<PatientEntrySaveClicked>()
-        .withLatestFrom(phoneNumberChanges, noPhoneToggles)
-        .map { (_, phoneNumber, isNoneSelected) ->
-          when {
-            (phoneNumber.isBlank() && !isNoneSelected) -> listOf(PHONE_NUMBER_EMPTY)
-            else -> listOf()
-          }
-        }
-
     val colonyChanges = events.ofType<PatientColonyOrVillageTextChanged>().map { it.colonyOrVillage }
-    val noColonyToggles = events.ofType<PatientNoColonyOrVillageToggled>().map { it.noneSelected }
 
     val colonyErrors = events
         .ofType<PatientEntrySaveClicked>()
-        .withLatestFrom(colonyChanges, noColonyToggles)
-        .map { (_, colony, isNoneSelected) ->
+        .withLatestFrom(colonyChanges)
+        .map { (_, colony) ->
           when {
-            (colony.isBlank() && !isNoneSelected) -> listOf(COLONY_OR_VILLAGE_EMPTY)
+            colony.isBlank() -> listOf(COLONY_OR_VILLAGE_EMPTY)
```

Comment:
Colony was previously validated in two places. Once here and inside `OngoingPatientEntry#validationErrors()` because the entry model did not know the existence of the none checkbox. Now that the checkbox is removed, I think this is no longer required.